css/main.scss

```
@@ -796,6 +850,109 @@ ol > p {
   align-items: center;
   justify-content: center;
 }
+.benefits-icons{
+  flex-wrap: nowrap;
+}
+.benefits{
+  display: flex;
+  justify-content: center;
+  width: 90%;
+  div{
+    padding: 0% 5%;
+  }
+  img{
+    padding:1% 5%;
+  }
+}
+.slider {
+  top: 50%;
+  left: 50%;
+  transform: translate(0%);
+  width: $slider-width;
+  height: $slider-height;
+}
+.wrapper {
+  overflow: hidden;
+  position: relative;
+  width: $slider-width;
+  height: $slider-height;
+  z-index: 1;
+}
+.slides {
+  display: flex;
+  position: relative;
+  top: 0;
+  left: -$slider-width;
+  width: 10000px;
+}
+.slides.shifting {
+  transition: left .2s ease-out;
+}
+.slide {
+  width: $slider-width;
+  height: $slider-height;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  transition: all 1s;
+  position: relative;
+}
+.control {
+  position: absolute;
+  top: 35%;
+  width: 50px;
+  height: 50px;
+  z-index: 2;
+}
+.prev,
+.next {
+  background-size: 22px;
+  background-position: center;
+  background-repeat: no-repeat;
+  cursor: pointer;
+}
+.prev {
+  background-image: url('../img/arrow_left.svg');
+  left: -100px;
+}
+.next {
+  background-image: url('../img/arrow_right.svg');
+  right: -100px;
+}
+.prev:active,
+.next:active {
+  transform: scale(.8);
+}
+blockquote{
+  position: relative;
+  text-align: left;
+}
+blockquote::after{
+  box-sizing: border-box;
+  content: "“";
+  position: absolute;
+  font-size: 17rem;
+  font-weight: bold;
+  z-index: -1;
+  left: 0rem;
+  top: -6.9rem;
+  color: #f2f3f4;
+}
```

@nikita-deriv  LGTM. just a small thing, can we nest this `blockquote`

```suggestion
blockquote {
  position: relative;
  text-align: left;
  
    &::after {
    box-sizing: border-box;
    content: "“";
    position: absolute;
    font-size: 17rem;
    font-weight: bold;
    z-index: -1;
    left: 0rem;
    top: -6.9rem;
    color: #f2f3f4;
  }
}

```

---

Thanks, fixed with your suggestion